notebooks/pdf_data_converter.ipynb

```diff
@@ -0,0 +1,264 @@
+{
```

Comment:
<p>This is already pretty awesome, thanks Surya! I guess we can add some text cleaning here to further improve it. How are we getting the context here? Maybe we could remove the header, the footer, and all the questions from the text and the remaining string can be one big context paragraph.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Thanks @Shreyanand . Context in this case is the whole content of the page that the answer is from. I am working on cleaning it up.

Reply:
<p>The context as page idea is good, we can remove questions and unnecessary text from the page and add that as context. </p>

Reply:
<p><em>Can I define a custom domain and certificate for my applications?</em></p><p><em>Yes. See Configuring custom domains for applications for more information.</em></p><p> </p><p>I was just looking at an output (ID 70) and it seems it is splitting at the full stop...</p>

Reply:
<p>Yes. I think we need to clean up a bit. Currently, it is at the basic step , e.g, question and answer were splitted by '?'. and Sentences were splitted by "." </p>